src/main/webapp/index-style.css

```diff
@@ -0,0 +1,81 @@
+#map {
+  margin: 0px;
```

Comment:
### nit
I just reread the CSS Style guide and saw that [Declaration Order should be alphabetical](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order). I have not been following this, either.
@mwo-2020, would you want us to update that in this PR or could be table it for a future PR?

Reply:
A follow-up PR is fine.